websites/S/Streaming+/presence.ts

```diff
@@ -0,0 +1,124 @@
+import { ActivityType, Assets, getTimestampsFromMedia } from 'premid'
+
+const presence = new Presence({
+  clientId: '1359510663159877642',
+})
+const getStrings = presence.getStrings({
+  playing: 'general.playing',
+  paused: 'general.paused',
+  watchingLive: 'general.watchingLive',
+  waitingLive: 'general.waitingLive',
+  waitingLiveThe: 'general.waitingLiveThe',
+  waitingVid: 'general.waitingVid',
+  waitingVidThe: 'general.waitingVidThe',
+  buttonWatchStream: 'general.buttonWatchStream',
+  buttonViewPage: 'general.buttonViewPage',
+  viewAPage: 'general.viewAPage',
+})
+const browsingTimestamp = Math.floor(Date.now() / 1000)
+
+let data: Metadata | undefined
+
+presence.on('UpdateData', async () => {
+  const strings = await getStrings
```

Comment:
To allow for users to quickly change their language, change `getStrings` into a function or directly embed `presence.getStrings`